assets/app/views/settings.html

```diff
@@ -65,22 +65,53 @@
                 </div>
 
                 <div ng-repeat="quota in quotas" class="gutter-bottom">
-                  <div ng-if="quote.status.hard.cpu || quota.status.hard.memory">
-                    <h2>
-                      Quota <span ng-if="(quotas | hashSize) > 1">{{quota.metadata.name}}</span>
-                    </h2>
-                    <div ng-if="$first" class="help-block">{{quotaHelp}}</div>
+                  <h2>
+                    Quota <span ng-if="(quotas | hashSize) > 1">{{quota.metadata.name}}</span>
+                  </h2>
+                  <div ng-if="$first" class="help-block">{{quotaHelp}}</div>
+                  <dl ng-if="quota.spec.scopes.length" class="left">
+                    <dt>Scopes:</dt>
+                    <dd>
+                      <div ng-repeat="scope in quota.spec.scopes">
+                        {{scope | humanizeQuotaScope}}
```

Comment:
what about just making the filter be 'startCase' since thats all it does? i'm sure we will use it for something else later